install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -111,6 +111,60 @@ spec:
 labels of targeted PV.
 ====
 
+[[selector-label-volume-bind]]
+=== Optional: Bind the PVC to a specific PV when using a selector
+
+Using a selector, a PVC can require a PV to have specific labels. But a PVC without any selector matches any PV.
+
+As the *cluster-admin* user, you can bind the PVC to a specific PV only.
+
+To reserve a PV for specific tasks, you have two options: create a specific storage class, or pre-bind the PV to your PVC.
+
+. Request a specific storage class for the PV by specifying the storage class’s name.
++
+The following resource shows the required values that you use to configure a StorageClass. This example uses the AWS ElasticBlockStore (EBS) object definition.
+
++
+.StorageClass definition for EBS
+====
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: gp2
```

Comment:
I would perhaps use a specific name, for example "kafka", so it's clear the storage class is reserved to it.

"gp2" is our default storage class that might confuse users.

Reply:
Changed `gp2` to `kafka`